### DISCUSSION TITLE
Issue Creating Course Category in Chamilo 1.11.26 - Fatal Error

### DISCUSSION THREAD

**Discussion by PoleNumeriqueCFASete** [chamilo/chamilo-lms#5205] [General]
Hello everyone,

I'm reaching out for assistance with an issue we're experiencing on our Chamilo platform, version 1.11.26. Whenever we attempt to create a new course category, we're met with a "Fatal error" message, which prevents us from completing the process.

Attached is a screenshot of the error message to provide a clearer picture of the problem.

Does anyone have an idea what might be causing this error and, more importantly, how to resolve it? Any help or suggestions would be greatly appreciated, as this issue is currently hindering our course management efforts.

Thank you in advance for your time and assistance.

Best regards,
Catarina CHAMBEU
CFA Sète

![Capture d’écran 2024-03-04 103645](https://github.com/chamilo/chamilo-lms/assets/160041101/d46b5e47-3cc1-4b7a-9b1e-641afbbb9556)

**Comment by jgnosolored** [comment]
Hi @PoleNumeriqueCFASete
 Check this
https://github.com/chamilo/chamilo-lms/discussions/5163

**Comment by PoleNumeriqueCFASete** [comment]
Thanks you, it worked!

Have a nice day.
